### DISCUSSION TITLE
Add Illuminance value in webui for Illuminance sensors (ZbBridge)

### DISCUSSION THREAD

**Discussion by midnight1979** [arendst/Tasmota#16406] [Ideas]
If sensor have "Illuminance" attribute - does it possible to output it value on main page of tasmota Zbbridge?

Like in temperature sensors for example, add some icon maybe but the icon is not required...

**Comment by s-hadinger** [comment]
I would like to have a more general way of enriching the UI with Berry. I prefer to leave the UI minimal for now.

**Reply by barbudor** [reply]
Isn't `web_sensor( )` callback enough ?

**Comment by midnight1979** [comment]
> I would like to have a more general way of enriching the UI with Berry. I prefer to leave the UI minimal for now.

But this improvement will be able on esp32 only?

**Reply by s-hadinger** [reply]
That's the plan

**Reply by sfromis** [reply]
Berry is indeed ESP32 only, so....

And do expect that larger improvements of Z2T will primarily be ESP32, as ESP8266 just does not have the capacity for extensive local processing.

**Reply by midnight1979** [reply]
Does it mean that support for esp8266 will be discontinued soon?

**Reply by sfromis** [reply]
Not likely. Just not the place to expect Z2T enhancements, except for small changes.
